repos/system_upgrade/el7toel8/actors/rpmtransactionconfigtaskscollector/libraries/scanner.py

```diff
@@ -8,7 +8,9 @@ def load_tasks_file(path, logger):
     if os.path.isfile(path):
         try:
             with open(path, 'r') as f:
-                return list(set([entry.strip() for entry in f.read().split() if entry.strip()]))
+                return list(
```

Comment:
Just a nitpick - what if line starts with ` #` (space + hash)? It should be treated as a comment but it ends up in the list.

Reply:
We can require there will not be space :)